modular_skyrat/modules/modular_vending/code/vending.dm

```diff
@@ -14,8 +14,38 @@
 	if(skyrat_contraband)
 		contraband += skyrat_contraband
 
+	/// Time to make clothes amounts consistent!
+	for (var/item in products)
+		if(products[item] < 5 && ispath(item, /obj/item/clothing) && skyrat_IsNotGameplay(item))
+			products[item] = 5
+	for (var/item in premium)
+		if(premium[item] < 5 && ispath(item, /obj/item/clothing) && skyrat_IsNotGameplay(item))
+			premium[item] = 5
+			
 	QDEL_NULL(skyrat_products)
 	QDEL_NULL(skyrat_premium)
 	QDEL_NULL(skyrat_contraband)
 	return ..()
 
+/obj/machinery/vending/proc/skyrat_IsNotGameplay(var/obj/item/clothing/clothing_path)
+	var/obj/item/clothing/clothing = new clothing_path()
+
+	if(TRAIT_CHUNKYFINGERS in clothing.clothing_traits)
+		return FALSE
+	if(TRAIT_FASTMED in clothing.clothing_traits)
+		return FALSE
+	if(TRAIT_QUICK_CARRY in clothing.clothing_traits)
+		return FALSE
+	if(TRAIT_QUICKER_CARRY in clothing.clothing_traits)
+		return FALSE
+	if(TRAIT_DEAF in clothing.clothing_traits)
+		return FALSE
```

Comment:
Here's something a little cleaner that would do the same thing:
```suggestion
	var/banned_traits = list(
		TRAIT_CHUNKYFINGERS,
		TRAIT_FASTMED,
		TRAIT_QUICK_CARRY,
		TRAIT_QUICKER_CARRY,
		TRAIT_DEAF,
	)
	for(var/trait in banned_traits)
		if(trait in clothing.clothing_traits)
			return FALSE
```

Reply:
Thanks! I'll do this, but I'll have the list be at the object level, cause I think creating a list for potentially 10+ items per vendor is kinda bad for performance.